frontend/src/metabase/visualizations/components/ChartSettings.jsx

```diff
@@ -75,6 +75,18 @@ class ChartSettings extends Component {
         return assocIn(this.props.series, [0, "card", "visualization_settings"], this.state.settings);
     }
 
+    getChartTypeName(){
+      console.log(this.props.series[0].card)
```

Comment:
LGTM except this `console.log`. I'd also probably do a `switch statement or `const display = ...` at the top, but not a big deal.